src/renderer/components/SettingsSection/Developer.tsx

```diff
@@ -1,53 +1,50 @@
-import React, { FC, useEffect, useState } from 'react';
-import { useSetting } from "common/settings";
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { useSetting } from 'common/settings';
 
 const SettingsItem: FC<{ name: string }> = ({ name, children }) => (
-    <div className="flex flex-row items-center justify-between py-3.5">
-        <p className="m-0">{name}</p>
-        {children}
-    </div>
+  <div className="flex flex-row items-center justify-between py-3.5">
+    <p className="m-0">{name}</p>
+    {children}
+  </div>
 );
 
-const index = (): JSX.Element => {
-    const [configDownloadUrl, setConfigDownloadUrl] = useSetting<string>('mainSettings.configDownloadUrl');
-    const [configDownloadUrlValid, setConfigDownloadUrlValid] = useState<boolean>(false);
+export const DeveloperSettings = (): JSX.Element => {
```

Comment:
Could you please keep unrelated changes out of a linting PR?

Reply:
That is a linting change. It fixes the function not being considered a react component, which triggers a lint rule regarding hooks.